index.js

```diff
@@ -2,19 +2,35 @@
 const path = require('path');
 const locatePath = require('locate-path');
 
+function isLastDirEqualTo(dir, compareDir) {
+	if (!compareDir) {
+		return false;
+	}
+
+	const splittedPath = dir.split('/');
+	const lastFolder = splittedPath[splittedPath.length - 1];
```

Comment:
Very important to use [path.basename()](https://nodejs.org/api/path.html#path_path_basename_path_ext) here to grab the last path component, instead of splitting like this.

1. Hard coding `/` as a separator will not work with Windows-style paths.
2. Even if you use [path.sep](https://nodejs.org/api/path.html#path_path_sep), the path may end with a separator, which is quite common in practice. That will cause the final array element to be an empty string `''`. Can't use that as a directory name!

Both of the above issues will be taken care of by `path.basename()`.

There is another problem, though. What if `dir` is the root directory? You will need to take care of that separately, as even `path.basename()` will return an empty string in that case.

Reply:
Thank you for your advice 👍